maven-archetype-plugin/src/main/java/org/apache/maven/archetype/mojos/CreateArchetypeFromProjectMojo.java

```diff
@@ -210,7 +210,7 @@ public class CreateArchetypeFromProjectMojo
     /**
      * The directory where the archetype should be created.
      */
-    @Parameter( defaultValue = "${project.build.directory}/generated-sources/archetype" )
+    @Parameter( defaultValue = "${project.build.directory}/generated-sources/archetype", property = "outputDirectory" )
```

Comment:
shouldn't this be the field name by default?

Reply:
no isn't

Reply:
I would like to add a prefix `archetype.` like in other parameters